sinc/octave_minimal.sinc

```diff
@@ -0,0 +1,6 @@
+apps:
+{
+    $CURDIR/../apps/octave/actions.${LEGATO_TARGET}.app
+    $CURDIR/../apps/octave/virtual.${LEGATO_TARGET}.app
+    $CURDIR/../apps/octave/cloudInterface.${LEGATO_TARGET}.app
+}
```

Comment:
Missing newline on last line.

Reply:
Is this relevant in 2019?  Not trolling.  I'm just curious.

Reply:
I don't think it makes much difference.  It just shows up as a red flag in a bunch of code review tools (including GitHub), so it looks nicer if there's a newline.  I don't really care as long as it works.  It's not a readability issue or anything.